cmd/ocm-backplane/cloud/common.go

```diff
@@ -0,0 +1,128 @@
+package cloud
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/arn"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/backplane-cli/pkg/awsutil"
+	"github.com/openshift/backplane-cli/pkg/utils"
+	"io"
+)
+
+const OldFlowSupportRole = "role/RH-Technical-Support-Access"
+
+var StsClientWithProxy = awsutil.StsClientWithProxy
+var AssumeRoleWithJWT = awsutil.AssumeRoleWithJWT
+var NewStaticCredentialsProvider = credentials.NewStaticCredentialsProvider
+var AssumeRoleSequence = awsutil.AssumeRoleSequence
+
+type assumeChainResponse struct {
+	AssumptionSequence []namedRoleArn `json:"assumptionSequence"`
+}
+
+type namedRoleArn struct {
+	Name string `json:"name"`
+	Arn  string `json:"arn"`
+}
+
+func getIsolatedCredential(clusterID string) (aws.Credentials, error) {
```

Comment:
I went with "credential" because in `credentials.go` line 148 of this PR, the existing function for the "old flow" was named "getCloudCredential". Would you still like me to change this?

Reply:
I'd be +1 to this change. There is no historical reason I can remember other than most likely me making a typo and that persisting forever :)

Reply:
Done, thanks!